### ISSUE TITLE
[Bug] New project has Typescript errors in tests.

### ISSUE THREAD

**Issue by kilogold** [magicblock-labs/bolt#106]
**Describe the bug**
The default `bolt test` runs successfully, but linter errors exist on the following default test:

Line 9:
```
Module '"@magicblock-labs/bolt-sdk"' has no exported member 'Program'. ts(2305)
```

Line 45:
```
Property 'entityId' does not exist on type '{ instruction: TransactionInstruction; transaction: Transaction; entityPda: PublicKey; }'. Did you mean 'entityPda'? ts(2551)
```

**To Reproduce**

1. `bolt init bug`
2. `cd bug`
3. `bolt test`

**Workaround**
- Manually import `Program` from Anchor package.
- Use `entityPda` instead of `entityId`.

**Expected behavior**
No linter errors.

**Additional context**
Using `bolt-cli 0.1.10`

**Comment by notdanilo** [comment]
@kilogold can you try the latest bolt-cli with `cargo install --git https://github.com/magicblock-labs/bolt bolt-cli`?
I will publish a new version asap.

**Comment by notdanilo** [comment]
It should be fixed in `0.1.11`. Please re-open the issue if it still happens.
